jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/upload/UpdateFunctionCodeDialog.kt

```diff
@@ -77,12 +78,22 @@ class UpdateFunctionCodeDialog(private val project: Project, private val initial
                 title = message("lambda.service_name"),
                 content = message("lambda.function.code_updated.notification", initialSettings.name)
             )
-            LambdaTelemetry.editFunction(project, update = false, result = Result.Succeeded)
+            LambdaTelemetry.editFunction(
+                project,
+                lambdaPackageType = LambdaPackageType.from(initialSettings.packageType.toString()),
+                update = false,
```

Comment:
This file is UndateFunctionCode, why is update false?

Reply:
Because this is how it was before :P, this has been changed to deploy which we should have been using before but wern't